extensions/config.py

```diff
@@ -181,21 +179,119 @@ async def config_moderation(ctx: lightbulb.Context):
             f"{'**CHANGED** ' if ctx.options.update_log.id != result['updates'] else ''}Bot Updates Channel",
             # pylint: disable=line-too-long
             f"<#{ctx.options.update_log.id}>", inline=True)
-    await ctx.respond(embed=embed, flags=EPHEMERAL)
+    await ctx.respond(embed=embed)
     await mydb.close()
 
 
-@CONFIG.command()
+@config.child
```

Comment:
```suggestion
@config.child
```
Just realized, this also pushes the initial testing for user inputted ban messages.